markets/perps-market/contracts/modules/LiquidationModule.sol

```diff
@@ -43,14 +54,43 @@ contract LiquidationModule is ILiquidationModule {
     }
 
     function _liquidateAccount(PerpsAccount.Data storage account) internal {
-        account.liquidateAccount();
+        uint128 accountId = account.id;
+        uint256[] memory openPositionMarketIds = account.openPositionMarketIds.values();
 
-        // TODO: account can be removed from liquidation if the margin reqs are met,
+        uint accumulatedLiquidationRewards;
 
-        if (account.openPositionMarketIds.length() == 0) {
-            GlobalPerpsMarket.load().liquidatableAccounts.remove(account.id);
+        for (uint i = 0; i < openPositionMarketIds.length; i++) {
+            uint128 positionMarketId = openPositionMarketIds[i].to128();
+            (uint256 amountLiquidated, int128 newPositionSize) = account.liquidatePosition(
+                positionMarketId
+            );
+
+            emit PositionLiquidated(accountId, positionMarketId, amountLiquidated, newPositionSize);
```

Comment:
Can we move this event to after the fees and liquidation rewards are distributed? Otherwise we can't track fees paid during liquidations.

Reply:
the account updated event will have the actual paid keeper fee.  emitting the reward here doesn't make sense since its not the actual # the keeper got paid